src/App.jsx

```diff
@@ -27,6 +27,25 @@ function App() {
         </Route>
       </Routes>
       <>
+        <section>
+          <h1 className="home-header">Bruin Dictionary</h1>
+          <p className="home-description">
+            Welcome to Bruin Dictionary! A crowdsourced online dictionary for UCLA’s acronyms, slang
+            words, and inside jokes.
+          </p>
+          <a href="#" className="home-button">
+            Continue
```

Comment:
what is this button for?

Reply:
Since this is in App.jsx, this is shown on all pages including login page, add page, etc

Reply:
looks good tho